tools/src/main/java/org/apache/kafka/tools/reassign/Tuple2.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.tools.reassign;
+
+import java.util.Objects;
+
+public final class Tuple2<V1, V2> {
+    public final V1 v1;
+
+    public final V2 v2;
+
+    public Tuple2(V1 v1, V2 v2) {
+        this.v1 = v1;
+        this.v2 = v2;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Tuple2<?, ?> tuple = (Tuple2<?, ?>) o;
+        return Objects.equals(v1, tuple.v1) && Objects.equals(v2, tuple.v2);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(v1, v2);
+    }
+
+    @Override
+    public String toString() {
+        return "Tuple{v1=" + v1 + ", v2=" + v2 + '}';
```

Comment:
What about using class name `Tuple2` at the `toString()` after rename the class?

Reply:
Fixed.